### PR TITLE
Bump react-devtools-inline from v4.4.0 to v4.13.2

### PR DESCRIPTION
**Pull request by iChenLei**
## What kind of change does this PR introduce?
Bump `react-devtools-inline` from v4.4.0 to v4.11.1

## What is the current behavior?
It work, but lack many new feature and encount some bugs, to see [react-devtools-inline CHANGELOG](https://github.com/facebook/react/blob/master/packages/react-devtools/CHANGELOG.md)

## What is the new behavior?
More useful feature and more bugfix. ( I contributed a bugfix https://github.com/facebook/react/pull/20534

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Step A
2. Step B
3. Step C

## Checklist

- [ ] Documentation
- [x] Testing 
- [x] Ready to be merged
      
- [ ] Added myself to contributors table

### CODE REVIEW

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5654/commits/c073feb16ff6fdd47f4197c2bf5e1aeac78620fb is at https://pr5654.build.csb.dev/s/new.

**Comment by iChenLei** [comment]
@lbogdan Hi, `https://pr5654.build.csb.dev/s/new` It looks like not work cause `CORS`, so i can't check and test `React Devtools` v4.11.1.

**Review by danilowoz — CHANGES_REQUESTED** [review]
Hey @iChenLei, in order to fix the CORS issue, try to use a Google Chrome extension that allows CORS, it worked for me!

However, it seems that it introduced a new bug in the React Devtool: 
Screenshot 2021-05-11 at 15 44 51

Want you to give it a shot and try to fix it? BTW, thanks for contributing!
Let me know if you need any help with this.

File: yarn.lock [hunk 1]
```diff
@@ -26440,12 +26440,10 @@ react-dev-utils@^9.0.0:
...
-react-devtools-inline@^4.4.0:
-  version "4.4.0"
-  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.4.0.tgz#e032a6eb17a9977b682306f84b46e683adf4bf68"
-  integrity sha512-ES0GolSrKO8wsKbsEkVeiR/ZAaHQTY4zDh1UW8DImVmm8oaGLl3ijJDvSGe+qDRKPZdPRnDtWWnSvvrgxXdThQ==
-  dependencies:
-    es6-symbol "^3"
+react-devtools-inline@^4.11.1:
+  version "4.11.1"
+  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.11.1.tgz#406cdc370fd416393a81a4654991956d8c7d6157"
+  integrity sha512-pOC3Rscr2S2D1l2NQuDYdC0WHaf1KjW/IB8nBoUtT5AUM1X1yfmbWFANzr1FeT21v4eF9FwjaVpDXnl2kSlRdg==
```
Comment (iChenLei) [inline yarn.lock:26446]:
```suggestion
react-devtools-inline@4.13.2:
  version "4.13.2"
  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.13.2.tgz#59a5991ab2a946acedd871693a4ceca3ed7f7441"
  integrity sha512-erlHuLtRTClO+B79eu4nyjHGt0vFak1UoDpmSNmA9brF5SsdUTLsmiv6huTnJyNJ2pDKF8nUaekXkRf6E7mwqg==
```

Comment (iChenLei) [inline yarn.lock:26446]:
```suggestion
react-devtools-inline@4.13.2:
  version "4.13.2"
  resolved "https://registry.yarnpkg.com/react-devtools-inline/-/react-devtools-inline-4.13.2.tgz#59a5991ab2a946acedd871693a4ceca3ed7f7441"
  integrity sha512-erlHuLtRTClO+B79eu4nyjHGt0vFak1UoDpmSNmA9brF5SsdUTLsmiv6huTnJyNJ2pDKF8nUaekXkRf6E7mwqg==
```

**Review by iChenLei — COMMENTED** [review]
update the version

File: yarn.lock (hunk 1 above)
Comment (danilowoz) [reply yarn.lock:26446]: Unfortunately, it's still happening the same issue :(
Are you able to run locally and make sure this isn't happening only in the deployment preview?

**Pull request closed without merging**